include/tvm/relax/transform.h

```diff
@@ -215,17 +215,108 @@ TVM_DLL Pass AnnotateTIROpPattern();
  */
 TVM_DLL Pass FuseOps(int fuse_opt_level = -1);
 
+/*!
+ * \brief The pattern object used as the input of FuseOpsByPattern. For bindings to be
+ * fused, it needs to be matched with `pattern` and the `check` function needs to return
+ * true.
+ */
+class FuseOpsPatternNode : public Object {
```

Comment:
The name `FuseOpsPattern` might be confusing, since it sounds more related to `FuseOps` than `FuseOpsByPattern`.

Reply:
As discussed offline, `FuseOpsPattern` is renamed to `FusionPattern` and `PatternCheckFunctionInput` is renamed to `PatternCheckContext`.